### ISSUE TITLE
file #180 is not written

### ISSUE THREAD

**Issue by Masterluke87** [psi4/psi4#918]
Dear all,

I try to obtain molecular orbitals for a converged calculation. I use the following input:
```python
set guess sad
set scf_type df
set reference rhf

psi4_io.set_specific_path(180, './')
psi4_io.set_specific_retention(180, True)

energy('scf')
```
and start psi4 with the "-m" flag. However, I do not get this file. Is there another command/way that I have to use?

thanks,
Christopher

**Comment by loriab** [comment]
File 180 is in transition right now from its longstanding form (which was under psio indexing and control and thus responsive to psi4_io commands) to a numpy version that's more suited to python interaction (and that's under a different retention control). For now you can get 180 by commenting out [this line](https://github.com/psi4/psi4/blob/master/psi4/extras.py#L45).

**Comment by Masterluke87** [comment]
Thanks for your answer! But can I use this file for a restart? This seems not to work:

`energy('scf',restart_file="./output.default.17274.180.npz")
` 

thanks,
Christopher

**Comment by dgasmith** [comment]
Your right, this should be patched up in #918. As a note you will need to add a `set guess read` in your input file for this to work.
